backend/src/main/java/org/example/backend/domain/orders/model/entity/Orders.java

```diff
@@ -0,0 +1,71 @@
+package org.example.backend.domain.orders.model.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.example.backend.domain.orders.model.dto.OrderDto.OrderCompleteRequest;
+import org.example.backend.global.common.constants.OrderStatus;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+public class Orders {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long idx;
+
+    private Long boardIdx;
+
+    private String ordersNumber; // 주문 번호
+
+    @Enumerated(EnumType.STRING)
+    private OrderStatus status;
+
+    @CreatedDate
+    private LocalDateTime createdAt;
```

Comment:
`@CreatedDate`를 쓸 때 `@Column(updatable = false)`를 함께 써줘야 데이터 업데이트할 때 null값이 들어가지 않을거라
`@Column(updatable = false)`도 추가해주시면 좋을거 같아요!

```suggestion
@CreatedDate
@Column(updatable = false)
private LocalDateTime createdAt;
```